### PR TITLE
data!

### PR DESCRIPTION
**Pull request by sandreae**

### CODE REVIEW

File: pubs-test.json [hunk 1]
```diff
@@ -31,22 +57,129 @@
...
+    ],
+    "tagsComposition":  [
+      { "source": "1", "target": "3"}
+    ]
+  }
+}
+
+
+
       ],
```
Comment (madwort) [inline pubs-test.json:183]: still got some crap down here at the end of the file...!?!

**Review by madwort — CHANGES_REQUESTED** [review]
looks like there's still issues?

**Comment by madwort** [comment]
(data looks good though!)

**Review by sandreae — COMMENTED** [review]
oh yeh! How do I make it so :-?

**Pull request merged**